webpack/assets/javascripts/react_app/components/common/ResourceSwitcher/index.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import {
+  Popover,
+  Button,
+  Icon,
+  ListGroup,
+  ListGroupItem,
+  OverlayTrigger,
+} from 'patternfly-react';
+import { ButtonToolbar } from 'react-bootstrap';
+import EllipsisWithTooltip from 'react-ellipsis-with-tooltip';
+import PropTypes from 'prop-types';
+import './ResourceSwitcher.scss';
+
+const ResourceSwitcher = ({ resources }) => {
+  const popover = (
+    <Popover id="popover-contained" title={__('Resource Switcher')}>
```

Comment:
What if we just omit the title?  If you look at the version in katello it doesn't have a title but instead has a search bar.

<img width="534" alt="screen shot 2018-02-20 at 2 25 44 pm" src="https://user-images.githubusercontent.com/4116405/36444670-f7fcdce4-1649-11e8-8ca9-756475bfa19c.png">